INSTALL.txt

```diff
@@ -66,10 +66,10 @@ The system can then be tested and installed using the following commands:
   % sudo make install
 
 On a computer with multiple cores it is normally beneficial to build
-the system in parallel by adding a "-j" option to the invocation
-of make:
+the system in parallel by adding a "-jN" option to the invocation
+of make with a suitable value for N (typically the number of cores):
```

Comment:
With so many processors employing "hyperthreading", perhaps this should say the number of independent threads.

Reply:
No it really should be the number of cores, because the hyperthreads don't manage to improve things.  On my 8 core macbook pro (with 2x HT, so 16 hyperthreads) there is a slowdown when using an N that is greater than the number of cores:

    % time make -j8
    real 0m29.600s
    user 3m20.911s
    sys  0m11.614s

    % time make -j16
    real 0m33.521s
    user 5m15.157s
    sys  0m17.411s

I also tried N=7 and N=9 and they are worse than N=8.